ElectronClient/gui/NoteContentPropertiesDialog.tsx

```diff
@@ -54,28 +93,69 @@ export default function NoteContentPropertiesDialog(props:NoteContentPropertiesD
 		props.onClose();
 	};
 
-	const createItemField = (key: string, value: number) => {
-		const labelComp = <label style={Object.assign({}, theme.textStyle, theme.controlBoxLabel)}>{keyToLabel[key]}</label>;
-		const controlComp = <div style={Object.assign({}, theme.textStyle, theme.controlBoxValue)}>{value}</div>;
+	const labelCompStyle = {
+		...theme.textStyle,
+		fontWeight: 'bold',
+		width: '10em',
+	};
+
+	const controlCompColumnWidth = '7em';
```

Comment:
Can we do without the hard-coded width?

Reply:
Line 99 is  actually based off `themes.js`, line 183. However, I could not use the `style.controlBoxLabel` because of the `inline-block` and the `margin-right`.

As of line 102, the width has been hardcoded to fit the hardcoded strings. However, since you mention it, I think it is not an appropriate thing considering internationalisation. After making changes in the titles as asked by @tessus, I think it looks okay with the width CSS removed and allowing the width to fit the content:
![Screenshot from 2020-04-14 19-02-21](https://user-images.githubusercontent.com/54888685/79230856-d9043000-7e82-11ea-9e35-0d1ab527af98.png)

If you want me to make some other changes to the styling, please tell me.